walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -75,24 +67,38 @@ image::images/artifacts.png[integration, role="integr8ly-img-responsive"]
 // To do that please follow README in https://github.com/aerogear/mobile-services-installer
 
 [time=20]
-== Create DataSync Showcase Server Application from OpenShift template
+== Deploying the TODO Data Sync server
```

Comment:
Everywhere else it will be described as Showcase - including artifacts/templates and app itself. I'm actually not sure how we should refer to this as a whole. 
It implements simple WFM tasks (when you use imagination :) )